modules/rpc/handler.go

```diff
@@ -34,6 +35,11 @@ type contextInboundMiddleware struct {
 }
 
 func (f contextInboundMiddleware) Handle(ctx context.Context, req *transport.Request, resw transport.ResponseWriter, handler transport.UnaryHandler) error {
+	stopwatch := stats.RPCHandleTimer.
+		Tagged(map[string]string{stats.TagProcedure: req.Procedure}).
+		Timer("rpc.handler." + req.Procedure + ".time").
```

Comment:
i think `.time` is extraneous, no? we already know it's a timer, but I'm a bit behind on the M3